src/test/pages/LecturePage.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { mount, shallow } from 'enzyme';
+import './loadSubject.mock';
+import { LecturePage } from '../../main/pages/LecturePage';
+
+describe('LecturePage', () => {
+    let renderedComponent;
+
+    const props = {
+        loadSubject,
```

Comment:
as for this ```loadSubject```, reporting:
```
 FAIL  src/test/pages/LecturePage.test.js
  ● LecturePage › encountered a declaration exception

    ReferenceError: loadSubject is not defined

       9 | 
      10 |     const props = {
    > 11 |         loadSubject,
         |         ^
```

I think you can keep it simple and just define here as ```loadSubject: jest.fn()```, and the same also for all the other functions that ```LecturePage``` uses/calls (like ```this.props.loadSubject```)